Doc/library/inspect.rst

```diff
@@ -292,6 +295,9 @@ attributes (see :ref:`import-mod-attrs` for module attributes):
 |                 |                   | created, or ``None``. See |
 |                 |                   | |coroutine-origin-link|   |
 +-----------------+-------------------+---------------------------+
+|                 | cr_suspended      | is the coroutine          |
```

Comment:
Actually, can you put this just after `cr_running` instead? so that `*_running|suspended` attributes are grouped.